src/include/users/complex.rb

```diff
@@ -26,6 +26,13 @@
 #		Jiri Suchomel <jsuchome@suse.cz>
 #
 # $Id$
+
```

Comment:
I would not include all these changes related to the integration of Y2Users and Yast::Users. I see this is useful for testing, but there is a separate PR for such an integration.

Reply:
Well, my goal is to make y2users branch working and @ancorgs then adapt it to whatever style fits. Also it is useful that it reports now issues, which really helps with feedback and testing.